ui.frontend/src/utils.js

```diff
@@ -233,24 +233,8 @@ class Utils {
      */
     static updateId(htmlElement, oldId, newId) {
         let elementWithId = htmlElement.querySelectorAll("#" + oldId)[0];
-        let errorElementWithId = htmlElement.querySelector("#" + `${oldId}__errormessage`);
```

Comment:
this needs to be there, this will break repeatability

Reply:
These methods are shifted to formfieldbase.js to handle repeatability